openlibrary/solr/process_stats.py

```diff
@@ -1,271 +0,0 @@
-"""Script to process various stats collected by the system and load them into solr.
```

Comment:
Let's make sure that the deleted code is ONLY extracting things from the db2 solr.  Any code that writes data to db should be preserved.